docs/schema/filterdescriptor-element-in-filterdescriptors-bdcmetadata-schema.md

```diff
@@ -180,11 +180,11 @@ The following sections describe attributes, child elements, and parent elements.
 </thead>
 <tbody>
 <tr class="odd">
-<td align="left"><p><span sdata="link"><a href="localizeddisplaynames-element-in-metadataobject-bdcmetadata-schema.md">LocalizedDisplayNames Element in MetadataObject (BDCMetadata Schema)</a></span></p></td>
+<td align="left"><p><span><a href="localizeddisplaynames-element-in-metadataobject-bdcmetadata-schema.md">LocalizedDisplayNames Element in MetadataObject (BDCMetadata Schema)</a></span></p></td>
```

Comment:
No need to change anything here, but another solution is to take out the "span" HTML tags.  both the `<span sdata="link">` and `</span>`, since once you delete the span formatting, it just becomes pointless code.  So this is another way of fixing this issue:
`<td align="left"><p><a href="localizeddisplaynames-element-in-metadataobject-bdcmetadata-schema.md">LocalizedDisplayNames Element in MetadataObject (BDCMetadata Schema)</a></p></td>`